satellite/dbcleanup/chore_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbcleanup_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestDeleteExpiredSerials(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		satellite := planet.Satellites[0]
+		node := planet.StorageNodes[0].ID()
+		satellite.DBCleanup.Chore.Serials.Pause()
+
+		var expiredSerials []storj.SerialNumber
+		for i := 0; i < 5; i++ {
+			expiredSerials = append(expiredSerials, storj.SerialNumber{byte(i)})
+		}
+
+		var freshSerials []storj.SerialNumber
+		for i := 5; i < 10; i++ {
+			freshSerials = append(freshSerials, storj.SerialNumber{byte(i)})
+		}
+
+		yesterday := time.Now().UTC().Add(-24 * time.Hour)
+		for i := 0; i < 5; i++ {
```

Comment:
nit:

```suggestion
		for i := 0; i < len(expiredSerials); i++ {
```

or use `range`.

If we change the number of elements appended above we won't need to update this line